impl/src/main/java/org/ehcache/impl/copy/SerializingCopier.java

```diff
@@ -27,6 +28,17 @@
   private final Serializer<T> serializer;
 
   /**
+   * Convenience method allowing to represent this copier's class as the expected type in configuration.
+   *
+   * @param <T> The type to work on
+   * @return the proper type
+   */
+  @SuppressWarnings("unchecked")
+  public static <T> Class<? extends Copier<T>> asCopierClass() {
+    return (Class) SerializingCopier.class;
```

Comment:
Right - I did back and forth on this change and left it with the wrong one at the end. Fixing.